lms/djangoapps/courseware/views/views.py

```diff
@@ -520,7 +521,7 @@ def course_about(request, course_id):
             professional_mode = modes.get(CourseMode.PROFESSIONAL, '') or \
                 modes.get(CourseMode.NO_ID_PROFESSIONAL_MODE, '')
             ecommerce_checkout_link = ecomm_service.checkout_page_url(professional_mode.sku)
-
+            ecommerce_bulk_checkout_link = ecomm_service.checkout_page_url(professional_mode.bulk_sku)
```

Comment:
@mattdrayer , Is it make effect if we have `None` in `professional_mode.bulk_sku` ? If yes, Then we should add a check above `524` 
```
If professional_mode.bulk_sku:
            ecommerce_bulk_checkout_link = ecomm_service.checkout_page_url(professional_mode.bulk_sku)
```